app/scripts/modules/core/src/pipeline/config/stages/findArtifactFromExecution/FindArtifactFromExecutionExecutionDetails.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+
+import { ExecutionDetailsSection, IExecutionDetailsSectionProps } from 'core/pipeline/config/stages/core';
+import { StageExecutionLogs, StageFailureMessage } from 'core/pipeline/details';
+
+export function FindArtifactFromExecutionExecutionDetails(props: IExecutionDetailsSectionProps) {
+  const { stage } = props;
+
+  return (
+    <ExecutionDetailsSection name={props.name} current={props.current}>
+      <div className="row">
+        <div className="col-md-9">
+          <dl className="dl-narrow dl-horizontal">
+            <dt>Match Artifact</dt>
+            <dd>{JSON.stringify(stage.context.expectedArtifact.matchArtifact)}</dd>
```

Comment:
I'm guessing this probably renders the JSON all on one line. You probably want `JSON.stringify(json, null, 2)`, or even better, you could use `jsonUtilityService.makeSortedStringFromObject(json)`, which does the same thing and also sorts the keys.

Reply:
ooh neat, thanks